### PR TITLE
remove code compression in karma

### PR DESCRIPTION
**Pull request by henrikrudstrom**
karma currently compresses the code under test, makes for bad stack traces...

### CODE REVIEW

**Comment by ChALkeR** [comment]
I don't think this works, you have to override that option only in `test-debug` (like the other karma options).

**Comment by henrikrudstrom** [comment]
@ChALkeR works here, but i can add it to the gulp task if you want, but then it should be added to test-watch as well, good to have a correct stacktrace there as well...

**Comment by ChALkeR** [comment]
@henrikrudstrom It broke the coverage reports, see the `[show all checks]` at this PR.

**Comment by henrikrudstrom** [comment]
thats a good argument... will update soon

File: karma.conf.js [hunk 1]
```diff
@@ -19,7 +19,7 @@ module.exports = function(config) {
       dir: 'coverage/'
     },
     preprocessors: {
-      'lib/qt.js': ['coverage']
+      'lib/qt.js': ['coverage'],
```
Comment (ChALkeR) [inline karma.conf.js]: unwanted edit

File: gulpfile.js [hunk 2]
```diff
@@ -76,7 +76,10 @@ gulp.task('test', ['build'], function(done) {
 
 gulp.task('test-watch', ['watch-dev'], function(done) {
   new karma.Server({
-    configFile: __dirname + '/karma.conf.js'
+    configFile: __dirname + '/karma.conf.js',
+    preprocessors: {
+      'lib/qt.js': []
+    },
```
Comment (ChALkeR) [inline gulpfile.js]: Do we want to disable code coverage generation in test-watch?

**Comment by henrikrudstrom** [comment]
good point, hopefully we can get both at the same time when we switch to browserify/webpack
@ChALkeR should be good to go now

File: gulpfile.js [hunk 3]
```diff
@@ -84,6 +84,9 @@ gulp.task('test-debug', ['watch-dev'], function(done) {
   new karma.Server({
     configFile: __dirname + '/karma.conf.js',
     browsers: ['PhantomJSCustom', 'Chrome'],
+    preprocessors: {
+      'lib/qt.js': []
+    },
```
Comment (ChALkeR) [inline gulpfile.js]: I just tested it, just `preprocessors: {},` also works. No need to specify the file.
Also, `reporters: ['progress']` would remove the generation of an empty coverage report.

**Comment by ChALkeR** [comment]
LGTM, merging.

**Comment by ChALkeR** [comment]
I'm squashing this.

**Pull request merged**